### ISSUE TITLE
Run conformance tests continuously against dcos/k8s cluster in CI

### ISSUE THREAD

**Issue by karlkfi** [mesosphere/kubernetes-mesos#592]

**Comment by sttts** [comment]
Blocked by
- lack of SSL support and service accounts
- lack of even manual working conformance tests
- lack of automation of DCOS deployment and testing

**Comment by sttts** [comment]
This has nothing to do with the MVP and its not feasible (see blockers above).

**Comment by karlkfi** [comment]
I completely disagree. I am not telling anyone to use k8s on mesos unless and until we have test automation.
